xcmailr-webapp/src/main/java/controllers/AdminHandler.java

```diff
@@ -448,19 +449,26 @@ public Result showEmailStatistics(Context context, @Param("dayPage") int dayPage
         return html;
     }
 
-    public Result getEmailSenderTablePage(Context context, @Param("scope") String scope, @Param("page") int page,
-                                          @Param("offset") int offset, @Param("limit") int limit,
-                                          @Param("sort") String sort, @Param("order") String order)
+    public Result getEmailSenderTablePage(Context context, @Param("scope") String scope,
+                                          @Param("page") Optional<Integer> page,
+                                          @Param("offset") Optional<Integer> offset,
+                                          @Param("limit") Optional<Integer> limit, @Param("sort") Optional<String> sort,
+                                          @Param("order") Optional<String> order)
```

Comment:
The parameter `page` is not used. Remove?

Are the remaining parameters really optional? The UI always sends a value for them.

Reply:
Sensible defaults can be used in case they are not given/valid. That's why I made them optional.

Reply:
All right, make sense. But please have another look at `page`, thx!

Reply:
Removed `page` parameter.